weave/trace_server/calls_query_builder.py

```diff
@@ -203,6 +218,37 @@ def as_select_sql(self, pb: ParamBuilder, table_alias: str) -> str:
         )
 
 
+class AggregatedDataSizeField(CallsMergedField):
+    def __init__(self, field: str = "total_size_bytes"):
```

Comment:
i'd recommend not having defaults here forcing the developer to define the field in the spec below